src/components/partnership/partnership.service.ts

```diff
@@ -473,15 +560,25 @@ export class PartnershipService {
         canEdit: !!result.canEditMouStatus,
       },
       mouStart: {
-        value: result.mouStart,
+        value: mouStart,
         canRead: !!result.canReadMouStart,
         canEdit: !!result.canEditMouStart,
```

Comment:
Where are these coming from?

Reply:
the partnership mou fields. not the overrides. I didn't see a reason to make more complex logic for this field. It behoves whoever does security for the partnership/project to ensure the user can read both the partnership and project mou fields.

Reply:
This field is calculated, not stored, so it doesn't make sense to have security for it.
`canRead` should be based on the override and project props security.
`canEdit` is always false.

Reply:
updates now check project mou perms.